public/scripts.js

```diff
@@ -33,6 +33,11 @@ document.addEventListener('DOMContentLoaded', function() {
       author.innerText = "– " + randomQuote.author;
       blockquote.classList.remove("flipOutX");
       blockquote.classList.add("flipInX");
+      
+      var quoteClass = quote.getAttribute("class")
+      if (quoteClass.indexOf("notation") === -1) {
+        quote.setAttribute("class", quoteClass + " notation")
+      }
```

Comment:
Actually, this can probably be simplified using the `quote.classList` property. For setting a class, you would do something like this:
```javascript
quote.classList.add(quoteClass + " notation");
```
Also, depending on which browsers we want to support for this, maybe consider using _Array.prototype.includes_ for the check? It reads a lot nicer, but [isn't that widely supported].

[isn't that widely supported]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes#Browser_compatibility